### PR TITLE
net-im/telegram-desktop: Bump to 3.1.0, openssl fixes

### PR DESCRIPTION
**Pull request by mid-kid**
@gyakovlev

### CODE REVIEW

File: media-libs/tg_owt/tg_owt-0_pre20210914.ebuild [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+	# Save about 15MB of useless headers
+	rm -r "${ED}/usr/include/tg_owt/base/third_party" || die
+	rm -r "${ED}/usr/include/tg_owt/common_audio/third_party" || die
+	rm -r "${ED}/usr/include/tg_owt/modules/third_party" || die
+	rm -r "${ED}/usr/include/tg_owt/third_party" || die
+
+	# Install third_party/libyuv anyway...
+	dodir /usr/include/tg_owt/third_party/libyuv/include
+	cd "${S}/src/third_party/libyuv/include"
```
Comment (gyakovlev) [inline media-libs/tg_owt/tg_owt-0_pre20210914.ebuild:114]: missing `die` here and on next line

Comment (mid-kid) [reply media-libs/tg_owt/tg_owt-0_pre20210914.ebuild]: Done, added it to the existing version of the ebuild, too.

**Comment by mid-kid** [comment]
@gyakovlev Ping? Not sure if you saw the fixes, just making sure.

**Comment by gyakovlev** [comment]
yeah, thanks, just busy. had in in merge queue for some time and finally pushed.

**Comment by mid-kid** [comment]
Ah. No worries. Thanks a ton.

**Comment by mid-kid** [comment]
Ah. No worries. Thanks a ton.

**Pull request closed without merging**